libs/tailwindcss-config/src/theme.js

```diff
@@ -16,69 +22,67 @@ module.exports = {
       yellow: '#EDFF22',
       pink: '#FF2D5E',
       green: '#00F780',
+      red: '#FF261A',
     },
     red: {
       DEFAULT: '#ED1515',
-      transparent: 'rgba(255, 38, 65, 0.3)',
-      vega: '#FF261A',
       dark: '#EB001B',
+      bar: 'rgba(47, 246, 139, 0.45)', // #2FF68B 45%
     },
     green: {
       DEFAULT: '#26FF8A',
-      transparent: 'rgba(38, 255, 138, 0.3)',
       dark: '#008545',
-      vega: '#00F780',
+      bar: 'rgba(47, 246, 139, 0.45)', // #2FF68B 45%
     },
     text: '#C7C7C7',
     deemphasise: '#8A9BA8',
     white: {
       DEFAULT: '#FFF',
-      '02': 'rgba(255, 255, 255, 0.02)',
-      '05': 'rgba(255, 255, 255, 0.05)',
-      10: 'rgba(255, 255, 255, 0.10)',
-      25: 'rgba(255, 255, 255, 0.25)',
-      40: 'rgba(255, 255, 255, 0.40)',
-      60: 'rgba(255, 255, 255, 0.60)',
-      80: 'rgba(255, 255, 255, 0.80)',
-      95: 'rgba(255, 255, 255, 0.95)',
-      100: 'rgba(255, 255, 255, 1.00)',
+      strong: '#FFF',
+      normal: '#F5F8FA',
+      muted: '#676767',
+      '02': shadeOfGray(2),
+      '05': shadeOfGray(5),
+      10: shadeOfGray(10),
+      25: shadeOfGray(25),
+      40: shadeOfGray(40),
+      60: shadeOfGray(60),
+      80: shadeOfGray(80),
+      95: shadeOfGray(95),
+      100: shadeOfGray(100),
     },
     black: {
       DEFAULT: '#000',
-      '02': 'rgba(0, 0, 0, 0.02)',
-      '05': 'rgba(0, 0, 0, 0.05)',
-      10: 'rgba(0, 0, 0, 0.10)',
-      25: 'rgba(0, 0, 0, 0.25)',
-      40: 'rgba(0, 0, 0, 0.40)',
-      60: 'rgba(0, 0, 0, 0.60)',
-      80: 'rgba(0, 0, 0, 0.80)',
-      95: 'rgba(0, 0, 0, 0.95)',
-      100: 'rgba(0, 0, 0, 1)',
+      strong: '#000',
+      normal: '#000',
+      muted: '#BFCCD6',
+      '02': shadeOfGray(100 - 2),
```

Comment:
Chore (blocking): A lot of these colours are shared with the black palette (i.e. white.40 is the same hex as black.60). As per the last comment, I think we should have a palette for solid greys, and we can create a ticket to assess where each palette is used

Reply:
It like this for purpose. It's just easier to use when you want to set up colors for dark and light themes
`<span className="text-black-95 dark:text-white-95">Text<span>`
instead of 
`<span className="text-black-95 dark:text-black-05">Text<span>`

It make it easier also to search and replace ie. text-(black|white)-95 -> text-$1-90